Backend/app/dashboard/provider-dashboard/src/API/ProviderPlatform/StaticOfferDriver/Ride.hs

```diff
@@ -134,12 +152,12 @@ rideInfo merchantShortId apiTokenInfo rideId = withFlowHandlerAPI $ do
   checkedMerchantId <- merchantAccessCheck merchantShortId apiTokenInfo.merchant.shortId
   Client.callBecknTransportBPP checkedMerchantId (.rides.rideInfo) rideId
 
-rideSync :: ShortId DM.Merchant -> ApiTokenInfo -> Id Common.Ride -> FlowHandler Common.RideSyncRes
-rideSync merchantShortId apiTokenInfo rideId = withFlowHandlerAPI $ do
+rideSync :: ShortId DM.Merchant -> ApiTokenInfo -> Common.RideSyncReq -> FlowHandler Common.RideSyncRes
+rideSync merchantShortId apiTokenInfo rideSynReq = withFlowHandlerAPI $ do
   checkedMerchantId <- merchantAccessCheck merchantShortId apiTokenInfo.merchant.shortId
-  transaction <- buildTransaction Common.RideSyncEndpoint apiTokenInfo rideId T.emptyRequest
+  transaction <- buildTransactionMutipleRides Common.RideSyncEndpoint apiTokenInfo rideSynReq.rideIds T.emptyRequest
   T.withResponseTransactionStoring transaction $
```

Comment:
Is this compiled? `transaction` have type `DT.TransactionMultipleRides`, but we need here `DT.Transaction`

Reply:
fixed it